sdk/python/feast/feature_server.py

```diff
@@ -265,6 +345,18 @@ def start_server(
         )
         monitoring_thread.start()
 
+    print("start_server called ")
```

Comment:
Maybe change these print statements to debug logs?

Reply:
done